app/src/main/res/values/colors.xml

```diff
@@ -6,4 +6,5 @@
     <color name="EmailTextAlpha">#B3FFFFFF</color>
     <color name="primaryBalticSea">#1E1D23</color>
     <color name="primaryCharade">#25252f</color>
+    <color name="backgroundDark">#1e1d23</color>
```

Comment:
This color is already defined on `line 7`.

Reply:
I have addressed it